UltiSnips/python.snippets

```diff
@@ -5,8 +5,19 @@ priority -50
 ###########################################################################
 
 #! header
-snippet #! "#!/usr/bin/env python" b
-#!/usr/bin/env python
+snippet #! "#!/usr/bin/env python3" b
+#!/usr/bin/env python3
```

Comment:
Can you make this one python with no number? The `#!3` snippet can be used for this.

Reply:
`#!` with no number is shortcut for using python3, because python3 is used more frequently now，just like https://github.com/honza/vim-snippets/blob/master/snippets/python.snippets.

Reply:
Yes but we wish to distinguish between the snippets rather than sharing triggers. `#!/usr/bin/env python` will run python3 on any up-to-date OS now.